src/main/java/com/intellij/compiler/notNullVerification/AnnotationThrowOnNullMethodVisitor.java

```diff
@@ -68,7 +68,9 @@ public AnnotationVisitor visitAnnotation(final String annotation, final boolean
     @Override
     @NotNull
     protected String getThrowMessage(int notNullParam) {
-        return "Argument " + getSourceCodeParameterNumber(notNullParam) + " for @NotNull parameter of " + className + "." + methodName + " must not be null";
+        int pnum = getSourceCodeParameterNumber(notNullParam);
+        String pname = parameterNames == null ? "" : " '" + parameterNames.get(pnum) + "'";
```

Comment:
I am sure it wont happen, but is there any possibility of arrayIndexOutOfBoundsError here while doing .get() ?